tests/symengine/step-44-sd-quadrature_level_01.cc

```diff
@@ -97,7 +97,7 @@ namespace Step44
 
       if (debug)
         {
-          SD::Utilities::print_substitution_map(std::cout, sub_vals);
+          SD::Utilities::print_substitution_map(deallog, sub_vals);
```

Comment:
The output here depends on a bool variable that is `false`. If someone was to change that, the test would then fail. Ultimately, it seems that this output is not intended to be recorded in an output file but only printed to the screen for debugging purposes.

Reply:
True. There are other places where these programs have `#ifdef DEBUG_EXTRA`. It is true that one could address all of these things, but that's more work than I'm willing to put in at the moment. I'm just trying to make things better, not optimal.

Reply:
Fair enough.